### ISSUE TITLE
Custom error messages (errorHandler)

### ISSUE THREAD

**Issue by juancarloscamargo** [OneIdentity/IdentityManager.Imx#160]
Hi,

I'm trying to have the StringErrors translated in my custom qer-app-portal. These are messages shown by the angular errorHandler.  In my case, I need to translate this:
![image](https://github.com/OneIdentity/IdentityManager.Imx/assets/12726971/245fc843-7e8c-4d6b-9581-de4806dd8070)

when a non authorized account tries to log in via oAuth.

I've tried the fix with the VI.Base.custom.vii  UTF-8 BOM placed in the bin folder of the apiserver:

`[en-US]
strError2329013="Necesita una cuenta protegida para acceder a este servicio."

[es-ES]
strError2329013="Necesita una cuenta protegida para acceder a este servicio."

[es]
strError2329013="Necesita una cuenta protegida para acceder a este servicio."

[default]
strError2329013="Necesita una cuenta protegida para acceder a este servicio."`

,  but it didnt seem to work  just as it did in the old web portal/applications. Is there any other workaround or should I extend the errorHandler with my own? 

Thanks!

**Comment by hannobunjes** [comment]
Hi,

I assume you are connected with an application server, please try putting the custom.vii file on the app server's bin directory.

**Comment by juancarloscamargo** [comment]
Hi,

Thanks. The apiserver is configured to connect via appserver, right. Placed the file in the bin directory of the app server , of the apiserver's too and restarted both, but it keeps showing the error in plain english. The file is saved with utf-8 bom , maybe I'm not capturing the right error number.

**Comment by NielsDeGroot** [comment]
Hello Juan Carlos,

Works for me with direct connection and over appserver.

[en-US]
strError810017="You shall not pass."

[nl-NL]
strError810017="Geen toegang."

[default]
strError810017="Wrong, try again."

The language specific entry also gets picked-up.

**Comment by juancarloscamargo** [comment]
Hi Niels!!
Thanks. I was in a hurry with a release and couldn't find a working solution, so eventually I've extended the ErrorHandler with a solution of my own. But I would like to stick to the "rule", so I'll retry again. 

My VI.Base.custom.vii UTF-8 BOM file is placed in the bin folder of both the apiserver and appserver. I'll doublecheck again and retry.  

All in all, it is a local problem for, as you both point out, it works. I'll close the issue later today.
Thanks again!
